main.js

```diff
@@ -10,3 +10,130 @@ document.querySelectorAll('.menu-link').forEach((n) => n.addEventListener('click
   hamburger.classList.remove('active');
   toolbarMenu.classList.remove('active');
 }));
+
+const popupSection = document.querySelector('.popupSection');
+const X = document.querySelector('.X');
+const page = document.querySelector('.page');
```

Comment:
- [ ] For the best name convention and JavaScript best practices, kindly rename your `X` to be `x` instead, lowercase is better to follow the vest practices.